engine/src/main/java/org/hibernate/search/engine/search/aggregation/dsl/RangeAggregationFieldStep.java

```diff
@@ -56,6 +58,17 @@ default <F> RangeAggregationRangeStep<?, PDF, F> field(String fieldPath, Class<F
 	 * See {@link ValueModel}.
 	 * @return The next step.
 	 */
-	<F> RangeAggregationRangeStep<?, PDF, F> field(String fieldPath, Class<F> type, ValueModel valueModel);
+	<F> RangeAggregationRangeStep<SR, ?, PDF, F> field(String fieldPath, Class<F> type, ValueModel valueModel);
+
+	/**
+	 * Target the given field in the range aggregation.
+	 *
+	 * @param fieldReference The field reference representing a <a href="SearchAggregationFactory.html#field-paths">path</a> to the index field to aggregate.
```

Comment:
I'm afraid linking to `#field-paths` won't be very helpful here.
Wouldn't it be better to add a section there about "static metamodel field references", with some generic info about what the static metamodel is and a recommendation to check it out in the reference documentation?